### ISSUE TITLE
Ref Frame as a Child of a Created Ref Frame Centers on the Sun

### ISSUE THREAD

**Issue by astrodavid10** [WorldWideTelescope/wwt-windows-client#170]
If I create a reference frame for an asteroid like Bennu and then create a ref frame as a child to show OSIRIS-REx's orbit at Bennu, it centers and scales for the sun instead. See the attached.
[OSIRIS-REx.zip](https://github.com/WorldWideTelescope/wwt-windows-client/files/5406409/OSIRIS-REx.zip)

**Comment by pkgw** [comment]
Do you think this is a new issue? This seems like the kind of functionality that would have been exercised in the past ..

**Comment by astrodavid10** [comment]
I don't think that it is new... I'm not sure I had ever put forth good effort to get a new ref frame to have children ref frames with something like a trajectory.

**Comment by pkgw** [comment]
Flagged by @astrodavid10 as something to look at for the upcoming release. CC @Carifio24.

**Comment by Carifio24** [comment]
@astrodavid10 I was looking into this using the example that you gave, but I'm unsure what I should expect to see at these reference frames. If I track Bennu, I see this:
![image](https://user-images.githubusercontent.com/14281631/152240988-3d111cc6-601f-42e4-ac0f-9e4529f60b4c.png)

When I track OSIRIS-REx, I see:
![image](https://user-images.githubusercontent.com/14281631/152241074-38d3da95-3d04-4228-bbf8-b1635cd03ab0.png)

I can pan around and see the trajectory intersecting itself, like this:
![image](https://user-images.githubusercontent.com/14281631/152241356-49460374-1a7e-4067-a5ca-9e3f60347e74.png)

I'm just curious what I should expect to see at the OSIRIS reference frame.

**Comment by astrodavid10** [comment]
The green are the orbits of OSIRIS-REx around Bennu and they should be attached to Bennu and not the Sun. If you are tracking OSIRIS-REx and zoom out you'll notice that it is residing in the Sun.

**Comment by astrodavid10** [comment]
@Carifio24 They are the correct shape and scale (or at least close) but are attached as a child of the sun incorrectly.

**Comment by Carifio24** [comment]
@astrodavid10 Ah, ok. For some reason, I'm actually unable to zoom in or out when tracking either of these layers. I also get the warning that "This file does not seem to be a valid collection", but the layers still appear in the layer manager.

I know that parts of WWTL files are often compressed, so I'm wondering if decompressing the .zip that you attached had some adverse effect. Could you share the (unzipped) layer file?
